webui/react/src/hooks/useModal/UserSettings/useModalCreateUser.tsx

```diff
@@ -53,7 +53,15 @@ const ModalForm: React.FC<Props> = ({ form, user, groups, viewOnly }) => {
 
   const updatePermissions = useCallback(async () => {
     if (user && canGetPermissions) {
-      const viewPermissions = await getUserPermissions({ userId: user.id });
+      const { roles } = await getUserPermissions({ userId: user.id });
+      const viewPermissions = new Array<Permission>();
```

Comment:
Little bit confused by this. This is the form for creating a user, right? Shouldn't we be selecting roles and not permissions?

Reply:
When we view or edit the user, this populates a viewable table of their permissions (a bit controversial especially in the future where they have permissions on specific workspaces)

The roles selector gets roles from `knownRoles` which comes from the Store and is updated by `fetchKnownRoles` on the UserManagement page.  It might be movable into this component if that seems better.

Reply:
If it's just for display then we really should be deriving those permissions in the render function but that's not important for this PR.